src/mechanics/timeIntegration/TimeIntegrationBase.cpp

```diff
@@ -101,16 +81,11 @@ void NuTo::TimeIntegrationBase::SetTimeDependentLoadCase(int rTimeDependentLoadC
     mTimeDependentLoadCase = rTimeDependentLoadCase;
```

Comment:
I thought we removed the load case. Should we clean it up here as well?
E.g. remove the member TimeIntegration::mTimeDependentLoadCase completely?